config/locales/devise.vi.yml

```diff
@@ -0,0 +1,63 @@
+vi:
```

Comment:
vi mà toàn tiếng anh thế này (yaoming)

Reply:
E dùng devise.en.yml của divise nên khi chuyển sanng vi thì vĩ lỗi 
Nên e copy nguyên file kia rồi sửa thành vi :)